docs/source/testing-and-debugging.md

```diff
@@ -43,71 +43,79 @@ from boa3.boa3 import Boa3
 Boa3.compile_and_save('path/to/your/file.py', debug=True)
 ```
 
-## Neo Test Runner
+## boa test constructor
 
 ### Downloading
 
-Install [Neo-Express](https://github.com/neo-project/neo-express#installation) and [Neo Test Runner](https://github.com/ngdenterprise/neo-test#neo-test-runner).
+Install [boa-test-constructor](https://pypi.org/project/boa-test-constructor/) with pip. We use this extension to run an isolated
+test environment for smart contracts with a neo-go node. When installing boa-test-constructor, [neo-mamba](https://dojo.coz.io/neo3/mamba/index.html)
+will be installed too.
```

Comment:
It is worth mentioning that the packages can be installed directly when neo3-boa is installed by using `pip install neo3-boa[test]`